rotkehlchen/tests/api/test_liquity.py

```diff
@@ -248,3 +251,26 @@ def test_account_with_proxy(rotkehlchen_api_server, inquirer):  # pylint: disabl
     # test that the list of addresses was not mutated
     rotki = rotkehlchen_api_server.rest_api.rotkehlchen
     assert len(rotki.chain_manager.accounts.eth) == 3
+
+
+@pytest.mark.parametrize('ethereum_accounts', [[JUSTIN, LIQUITY_POOL_DEPOSITOR]])
+@pytest.mark.parametrize('ethereum_modules', [['liquity']])
+@pytest.mark.parametrize('start_with_valid_premium', [True])
+def test_stability_pool(rotkehlchen_api_server):
+    """Test that we can get the status of the deposits in the stability pool"""
+    async_query = random.choice([False, True])
+    response = requests.get(api_url_for(
+        rotkehlchen_api_server,
+        'liquitystabilitypoolresource',
+    ), json={'async_query': async_query})
+    if async_query:
+        task_id = assert_ok_async_response(response)
+        result = wait_for_async_task_with_result(rotkehlchen_api_server, task_id)
+    else:
+        result = assert_proper_response_with_result(response)
+
+    assert JUSTIN in result
```

Comment:
what if Justing removes his lp? The only real way to make such a test is to mock the response